config/shelley-testnet.json

```diff
@@ -0,0 +1,22 @@
+{
+  "network": {
+    "protocolMagic": 764824073,
+    "backendUrl": "https://iohk-mainnet.yoroiwallet.com",
```

Comment:
We need the correct URL for the backend before we can release @vantuz-subhuman  @rcmorano

Reply:
https://shelley-testnet-yoroi-backend.yoroiwallet.com/api/ :)

Reply:
@rcmorano thanks! I’m updating it in another PR